web-jehlomat/src/screens/SetNewPassword/Components/index.ts

```diff
@@ -0,0 +1,6 @@
+﻿export * from "./Buttons";
```

Comment:
imho: this is a very common approach, but not sure it makes a lot of sense. Mostly, it is used when a project includes a lot of internal `packages`. So this is an entry point to export "public" components. So I personally do not think it is necessary here, bcz:
1. It will require to maintain, and it is not so easy, bcz all components are importable from every projects file and there is no eslint rule to detect it
2. VSCode/TypeScript auto-import will suggest 2 options, instead of one.
but it is up to you.

Reply:
Well, it isn't about a number of internal packages (if we're talking about NPM packages) but most likely about how to structure imports. It has some advantages like fewer boilerplates, but also some cons.
If you would keep it up to me, then I would keep it as it is now.